src/coreclr/vm/wasm/helpers.cpp

```diff
@@ -437,6 +438,12 @@ namespace
         (*fptr)(ARG(0), ARG(1), ARG(2));
     }
 
+    void CallFunc_I32_I32_I32_I32_I32_I32_RetVoid(PCODE pcode, int8_t *pArgs, int8_t *pRet)
```

Comment:
What is the method that this is needed for? I suspect that it may not be needed anymore after the simplification suggested above that creates the stubs lazily.

Reply:
This is needed for `RuntimeTypeHandle_GetActivationInfo`

Reply:
This is needed for `extern "C" void QCALLTYPE RuntimeTypeHandle_GetActivationInfo(...)`
https://github.com/dotnet/runtime/blob/cecdae61d396bf390ff6e2e5378dc2f804fb8606/src/coreclr/vm/runtimehandles.h#L148-L154